frontend-project/src/pages/channels/new/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { Button, Col, Card, Form, Input, Row } from 'antd';
+import { connect } from 'dva';
+import React, { useEffect, FunctionComponent } from 'react';
+import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
+
+interface ChannelNewFormProps {
+  name: String;
+  dispatch: Function;
+}
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const ChannelNewForm: FunctionComponent<ChannelNewFormProps> = () => {
```

Comment:
Chyba używaliśmy FC (jest to dokładnie ten sam interfejs ale wolałbym to w miarę unifikować)

Reply:
Używaliśmy obu: https://github.com/watchdogpolska/small_eod/search?q=FunctionComponent

Wystawię PR na ujednolicenie.